pkg/unpack/unpacker.go

```diff
@@ -240,7 +240,14 @@ func (u *Unpacker) unpack(
 	// TODO: Support multiple unpacks rather than just first match
 	var unpack *Platform
 
-	imgPlatform := platforms.Normalize(ocispec.Platform{OS: i.OS, Architecture: i.Architecture})
+	imgPlatform := platforms.Normalize(ocispec.Platform{
```

Comment:
This can just be `i.Platform` now, I think?

Reply:
Or is that not imported yet?

Reply:
Yeah, that's https://github.com/containerd/containerd/pull/7382 :sweat_smile:

I guess I could import that change and take over from @thaJeztah if you'd like me to? (he seems busy enough that I bet he wouldn't mind :joy:)